lockup-factory/src/lib.rs

```diff
@@ -0,0 +1,423 @@
+mod types;
+mod utils;
+
+use crate::utils::*;
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::{env, ext_contract, near_bindgen, AccountId, Promise, PromiseOrValue, Balance};
+use near_sdk::json_types::{U128};
+use near_sdk::serde::{Serialize};
+pub use crate::types::*;
+
+/// There is no deposit balance attached.
+const NO_DEPOSIT: Balance = 0;
+const TRANSFERS_STARTED: u64 = 1603274400000000000;
```

Comment:
This should be `1602614338293769340` from `near view transfer-vote.near get_result`
```suggestion
/// The block timestamp when the transfers were enabled on the mainnet.
/// To verify run `near view transfer-vote.near get_result`
const TRANSFERS_STARTED: u64 = 1602614338293769340;
```

Reply:
fixed